addons/website/static/src/js/menu/new_content.js

```diff
@@ -99,6 +100,38 @@ var NewContentMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
             return new Promise(function () {});
         });
     },
+    /**
+     * Asks the user information about a new channel to create, then creates it
+     * and redirects the user to this new channel.
+     *
+     * @private
+     * @returns {Promise} Unresolved if there is a redirection
+     */
+    _createNewChannel: function () {
```

Comment:
to move in website as previously

addons/website_livechat/static/src/js/website_livechat.editor.js